### DISCUSSION TITLE
Download error

### DISCUSSION THREAD

**Discussion by Heny67** [aclap-dev/video-downloadhelper#92] [Help and Q&A] [answered]
### Your question

Hi!
The program does not download to the directory specified in the settings, but says that it is not set and creates it.
The save location set in the settings:/home/en/Videos
But it's actually saved here:/proc/5511/cwd/Videos

Video DownloadHelper 8.1.4.0
Video DownloadHelper
Verzió: 8.1.4.0
Browser language: hu
Produkciós build
Build date: 2024-01-04
Build opciók: browser=firefox, linuxlic=false, noyt=false
Device information: Linux x86-64
Browser: Mozilla Firefox 122.0.1
We found the companion application: VdhCoApp 2.0.10
Companion app path: /home/en/vdhcoapp-2.0.10/vdhcoapp

**Comment by paulrouget** [comment]
Please update the addon and the coapp.

> But it's actually saved here:/proc/5511/cwd/Videos

Hm? Within /proc? That doesn't seem like a possible thing (proc is not writable afaik). Are you sure about that?

**Comment by Heny67** [comment]
Yes, but the directory name between the proc and cw directory changes after every download.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/80578702/c533f7c1-7b37-44fd-85b8-5f90c5ebc9cd)


The white rectangle says: The folder does not exist.
The "home/en/Videos" folder does not exist. Create it?
It can be seen framed that the missing directory exists.

**Reply by paulrouget** [reply]
Oh I see.

Hmm. Maybe it's an issue with the special character. If you just write "Video" (not "Videòk)" does it work?

**Comment by Heny67** [comment]
I think the problem will be somewhere here. Because it only sees that much of my 20 libraries. For example: neither the video.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/80578702/6cfbf072-0b04-4685-a419-b472a8054c2d)

**Reply by paulrouget** [reply]
Alright. I think it's a special character issue. I'll look into it.

**Comment by Heny67** [comment]
By the way, I created the same directory without accented characters, but it doesn't appear in the picture either, so I think the problem is somewhere else. By the way, I have been using the program for years without any problems, but this error has only appeared recently.

**Reply by paulrouget** [reply]
@Heny67 thank you for your patience. Can you try our beta?

Do this:

- make sure to have at least [CoApp version 2.0.17](https://www.downloadhelper.net/w/CoApp-Installation);
- install the [Beta version of the addon](http://www.downloadhelper.net/beta);
- Go to the settings of the addon, and enable "native filepicker":

<img width="593" alt="Screenshot 2024-02-16 at 20 43 20" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/dd8ef889-d526-4067-9b4a-1978269e9b5c">

---

Can you tell if this solves your issue?

**Comment by Heny67** [comment]
Hi!
There are so many changes that now I can see the videos folder, but not the downloaded file. If I select "open the location of the downloaded file" nothing happens.
It still saves the file here:/proc/5926/cwd/Videos

![image](https://github.com/aclap-dev/video-downloadhelper/assets/80578702/e1a0dcf9-d617-4d3a-9a21-289a9a610a7c)

**Reply by paulrouget** [reply]
You're doing this with the beta?

**Reply by paulrouget** [reply]
/proc/…/cwd/ is just an alias to another directory.

To be clear, what you do it this, correct?

- in the settings, in behavior, you change your download directory to `/home/en/Videos`
- **you click SAVE**
- them, when you download the video, you select "Download" (not quick download)
- then the video downloads

And you're saying it doesn't download under `/home/en/Videos` but under `/proc/…/cwd/Videos`?

The file doesn't exist under `/home/en/Videos`.

Correct?

**Comment by Heny67** [comment]
The answer to both of your questions is yes. If you look at the picture vdhcoapp 2.0.10. I downloaded both the 64-bit deb package and the tar package, and the 2.0.17 you suggested does not start. However, when installed from the command line, I get this back: "/vdhcoapp-2.0.17$ ./vdhcoapp
vdhcoapp is running successfully. This is not intended to be used directly from the command line. You should press Ctrl+C to exit. If your browser is unable to detect the coapp, run: "vdhcoapp install".
It's as if he were to wear it, but he doesn't use it.

"them, when you download the video, you select "Download" (not quick download)"
i never use fast download

"then the video downloads"
the video downloads, just to the wrong place.

**Reply by paulrouget** [reply, marked as answer]
Ah, that might be your issue.

Run the coapp like this:

```
./vdhcoapp install
```

And tell me if the coapp is recognized.

**Comment by Heny67** [comment]
Everything works fine now!
Thanks for the help!

**Reply by paulrouget** [reply]
Yeah!
